scripts/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# CMake build file for script files
+#
+# Copyright 2023 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+#
+
+add_subdirectory(common)
+
+if(DPDK_TARGET)
+    add_subdirectory(dpdk)
+elseif(ES2K_TARGET)
+    add_subdirectory(es2k)
```

Comment:
Nothing for Tofino target? Might want to add a `TODO` comment if its to be done later

Reply:
The Tofino IPU does not have any scripts. There are no plans to add any.